src/StatsBase.jl

```diff
@@ -146,6 +146,7 @@ module StatsBase
     indicatormat,   # construct indicator matrix
 
     # statistical models
+    PValue,
```

Comment:
Probably better not export this very small type until we have a real need for it.